cmd/check.go

```diff
@@ -25,4 +27,28 @@ func doCheck(ctx *cli.Context) error {
 	default:
 		return errors.Errorf("invalid storage type %v", config.StorageType)
 	}
+
+	return verifyNewUIDMap(ctx)
```

Comment:
Thanks for doing this.

Just move this above the switch(), and make it "if err := verifynewUIDMap(ctx); err != nil { return err };